### PR TITLE
Create ben10.ms

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@plr = player()
+
+
+
+
+@hero = associative_array(
+            'name': '§2B§fe§2n§010
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: missing a '

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            'class': 'ben10',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: classname should be benten

**Comment by MrCraft888** [comment]
done

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            'trail': '',
+            'op': 'false',
+            'speech': "Since when do I ever mess things up?!",
+            'powers': array(
+                "/buff",
+                "/jump 0-4",
+                "/run 0-6",
+                "/haste 0-3",
+                "/disguise",
+                "/disguiseviewself",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: this isn't a power

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            ),                            
+            'trail': '',
+            'op': 'false',
+            'speech': "Since when do I ever mess things up?!",
+            'powers': array(
+                "/buff",
+                "/jump 0-4",
+                "/run 0-6",
+                "/haste 0-3",
+                "/disguise",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: require evidence of disguising as other people and animals

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                "/disguiseviewself",
+                "/undisguise"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: proof required for regen

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                "/effect ". @plr ." 11 9999 3 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: proof required for resistance

**Comment by malonnnn** [comment]
file name needs to match the class name

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms (hunk 4 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: https://www.youtube.com/watch?v=z5nZUVCCy0E

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms (hunk 6 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: https://www.youtube.com/watch?v=XlOpga1p5gY

File: CommandHelper/LocalPackages/commands/buffs/ben10.ms (hunk 5 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/ben10.ms]: http://img08.deviantart.net/2333/i/2015/353/a/b/swampfire_s_regeneration__ben_10__by_dlee1293847-d9kqixn.png

File: CommandHelper/LocalPackages/commands/buffs/benten.ms [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            ),                            
+            'trail': '',
+            'op': 'false',
+            'speech': "Since when do I ever mess things up?!",
+            'powers': array(
+                "/buff",
+                "/jump 0-4",
+                "/run 0-6",
+                "/haste 0-3",
+                "/disguise",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/benten.ms]: proof required of disguising as people and animals

File: CommandHelper/LocalPackages/commands/buffs/benten.ms [hunk 8]
```diff
@@ -0,0 +1,34 @@
+@plr = player()
+
+
+
+
+@hero = associative_array(
+            'name': '§2B§fe§2n§010',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/benten.ms]: there should be a space between Ben and 10

File: CommandHelper/LocalPackages/commands/buffs/benten.ms [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+            'speech': "Since when do I ever mess things up?!",
+            'powers': array(
+                "/buff",
+                "/jump 0-4",
+                "/run 0-6",
+                "/haste 0-3"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/benten.ms]: requires proof of regen

File: CommandHelper/LocalPackages/commands/buffs/benten.ms [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+                "/effect ". @plr ." 11 9999 3 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/benten.ms]: requires proof of how much damage he can take. otherwise set to resist 1

**Pull request closed without merging**